tensilelite/Tensile/SolutionStructs/Solution.py

```diff
@@ -1604,6 +1612,7 @@ def calcLdsPad(lrvw: int, isaInfoMap: Dict[str, IsaInfo]) -> int:
               optPadA *= 2
             if readRegsB == 4 or readRegsB == 1:
               optPadB *= 2
+
         if ldsPadA == -1:
           if not state["UnrollMajorLDSA"]:
```

Comment:
This LdsPad calculation for UnrollMajorLDS=False is not always applicable to DTL.
As long as MT0(for A)/MT1(for B) * bpe is larger than or equal to DLT load unit (per 64 threads), this should work as non DTL.
If not, not sure but LDSPad might be useless if MT0(for A)/MT1(for B) * bpe is less than DLT load unit.

Reply:
Yeah, I think you are right, I don't know if lds padding (for DTL) would be helpful for `UnrollMajorLDS=False`. I've set it to zero (as you suggested), and added a note to check if there are cases where it may help.